tests/trainer/test_dataloaders.py

```diff
@@ -99,13 +125,8 @@ def test_dataloader_config_errors_init(tmpdir, dataloader_options):
 def test_multiple_val_dataloader(tmpdir):
```

Comment:
I'd suggest removing this test entirely... It's not testing anything useful really

Reply:
yeah.. must be very old.